provider/cldr/src/cldr_paths.rs

```diff
@@ -96,6 +98,9 @@ impl CldrPaths for CldrPathsLocal {
     fn cldr_misc(&self) -> Result<PathBuf, Error> {
         self.cldr_misc.clone().map_err(|e| e.into())
     }
+    fn uprops(&self) -> Result<Option<PathBuf>, Error> {
```

Comment:
Nit: This should return `Result<PathBuf, Error>`, where the error is `MissingSourceError` if not present

Reply:
Custom error makes more sense, missing source is if we have a path but the file is not there.